apps/dashboard/src/main/java/com/akto/utils/RiskScoreOfCollections.java

```diff
@@ -246,4 +247,37 @@ public void mapSensitiveSTIsInApiInfo(int timeStampFilter,int cronTime){
         }
     }
 
+    public void calculateRiskScoreForAllApis() {
+        int timeStamp = Context.now() - 24*60*60;
+        int limit = 1000;
+        int count = 0; 
+
+        List<WriteModel<ApiInfo>> bulkUpdates = new ArrayList<>();
+        Bson filter = Filters.or(
+            Filters.not(Filters.exists(ApiInfo.LAST_CALCULATED_TIME)),
```

Comment:
Check if this is using index. If not simplify using 
filters.and(filters.exists(false), Filters.gt() )